itests/util/src/main/java/org/apache/hadoop/hive/ql/QOutProcessor.java

```diff
@@ -353,10 +354,17 @@ private ArrayList<Pair<Pattern, String>> initPatternWithMaskComments() {
       {
         add(toPatternPair("(pblob|s3.?|swift|wasb.?).*hive-staging.*",
             "### BLOBSTORE_STAGING_PATH ###"));
-        add(toPatternPair(PATH_HDFS_WITH_DATE_USER_GROUP_REGEX, String.format("%s %s$3$4 %s $6%s",
-            HDFS_USER_MASK, HDFS_GROUP_MASK, HDFS_DATE_MASK, HDFS_MASK)));
+        add(toPatternPair(PATH_HDFS_WITH_DATE_USER_GROUP_REGEX, String.format("%s %s$3%s %s $6%s",
+            HDFS_USER_MASK, HDFS_GROUP_MASK, HDFS_SIZE_MASK, HDFS_DATE_MASK, HDFS_MASK)));
         add(toPatternPair(PATH_HDFS_REGEX, String.format("$1%s", HDFS_MASK)));
         add(toPatternPair("(.*totalSize\\s*=*\\s*)\\d+\\s*(.*)", "$1#Masked#$2"));
+        add(toPatternPair("(.*HDFS_BYTES_READ:*\\s*)\\d+\\s*(.*)", "$1#Masked#$2"));
```

Comment:
i am not sure we should mask HDFS_BYTES_READ, CACHE_HIT_BYTES, CACHE_MISS_BYTES completely. We might miss issues in the future because of that

Reply:
Considering if we should not mask it due to possibility of not detecting the issues, then we can only overwrite this for upgrades.

Reply:
that's ok

Reply:
I have updated the PR, please review it.
Thanks